src/appProvider.js

```diff
@@ -20,7 +20,7 @@ export const AppProvider = observer(({ xstore, store }) => (
   <Provider store={store}>
     <MobxProvider store={xstore}>
       <MuiThemeProvider theme={bodhiTheme}>
-        <IntlProvider locale={xstore.ui.locale} messages={xstore.ui.localeMessages}>
+        <IntlProvider locale={xstore.global.locale} messages={xstore.global.localeMessages}>
```

Comment:
can we please leave the `PubSubStore` and the `UiStore` separate?
[UiStore Example](https://mobx.js.org/best/store.html)
![image](https://user-images.githubusercontent.com/5455859/42188731-10c2bbb8-7e0a-11e8-8c81-92999ef1ede6.png)

The PubSubStore is used to modify other stores when a new block comes in. For instance, let's say we're on the `QTUM Prediction` page, and we get some new data, we then call `this.app.qtumPrediction.init()` from the `PubSubStore`. These are just 2 very separate concerns.

Reply:
We can also put `Error` in the `UiStore` since it's the `state of a global overlay`

Reply:
If we begin to have more and more realtime things happen with our app, this `GlobalStore` is going to get pretty long. I can show you an example from my last company if you'd like